packages/bsp/common/usr/lib/armbian/armbian-resize-filesystem

```diff
@@ -13,208 +13,287 @@
 
 do_expand_partition()
 {
-	# check if growroot (from cloud-initramfs-growroot package) is installed.
-	# despite it's name, that package does NOT require cloud-init.
-	# if so, it means the partition with root filesystem was already resized during initramfs.
-	# in this case do nothing here, but return 0 to allow resize2fs to run (growroot does not handle that).
-	if [[ -f /usr/share/initramfs-tools/hooks/growroot ]] || [[ -f /usr/share/initramfs-tools/scripts/local-bottom/growroot ]]; then
```

Comment:
*Please* don't remove this code, that keeps it compatible with growroot, which does the partition growing during initramfs. It would introduce bugs with all UEFI families. Thanks.

Reply:
i do that in line 277.

i check the source of `growroot`, it seem like that it doesn't support `crypt`. So i add a check